<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallKfxDialog</class>
 <widget class="QDialog" name="InstallKfxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Window Title">Install KeeperFX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/res/img/horny-face.png</normaloff>:/res/img/horny-face.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string comment="Header">Download and Install KeeperFX</string>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
 color: #AAA;
}</string>
     </property>
     <property name="text">
      <string comment="Information Label">Select your version, and this installer will download and install KeeperFX.

Choose the stable release for reliability or the alpha version if you want to help us out with testing.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="versionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
 color: #AAA;
}</string>
        </property>
        <property name="text">
         <string comment="Label">Release version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="versionComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="logTextArea">
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string comment="Button">Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditClear"/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="installButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string comment="Button">Install</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::SoftwareUpdateAvailable"/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
